tox.ini

```
@@ -150,12 +172,40 @@ deps =
 commands =
     python -VV
     mkdir -p {toxinidir}/_build/docs
-    sphinx-build -b html -d _build/docs/doctrees docs _build/docs/html
-#    sphinx-build -b doctest docs _build/docs/doctrees
+    sphinx-build -b html -D html_theme=alabaster -d _build/docs/doctrees docs _build/docs/html
```

I much prefer `nature` or `classic` over the lack of navigability in `alabaster`.

```suggestion
    sphinx-build -b html -D html_theme=nature -d _build/docs/doctrees docs _build/docs/html
```

---

The theme is irrelevant as the documentation is integrated in new Plone 6 documentation. So this can stay as before: **default** Alabaster. After merging of this PR, the documentation preview build will change to what is now 'tox -e plone6docs' which of course has **'sphinx-book-theme'**.